eland/dataframe.py

```diff
@@ -1376,6 +1374,8 @@ def to_json(
         --------
         :pandas_api_docs:`pandas.DataFrame.to_json`
         """
+        if index is None and PANDAS_VERSION[0] == 1:
+            index = True  # switch to the pandas 1 default
```

Comment:
The problem with that approach is that it's impossible to explicitly set `index` to `None`. I think it's an acceptable compromise. If someone complains, we can switch to a sentinel object instead to detect explicit `None`.

Reply:
That sounds okay, and automatically is no longer an issue once eland drops Pandas 1 support.